distro/travis/before_install.sh

```diff
@@ -34,20 +36,17 @@ elif [ "$TRAVIS_OS_NAME" = "osx" ]; then
 
   brew tap homebrew/python
```

Comment:
You can remove `homebrew/python`. All the formulae migrated to `homebrew/core` or `homebrew/science`.